src/test/java/seedu/address/testutil/WeightBuilder.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.testutil;
+
+import seedu.address.model.person.Weight;
+
+public class WeightBuilder {
```

Comment:
Similar to my comments above, seems to be unnecessary class to build a Weight object this way.